purchase_lot/views/purchase_order_view.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+        <record id="purchase_order_lot_form" model="ir.ui.view">
+            <field name="model">purchase.order</field>
+            <field name="inherit_id" ref="purchase.purchase_order_form" />
+            <field name="arch" type="xml">
+                <xpath
+                expr="//field[@name='order_line']/tree/field[@name='name']"
+                position="after"
+            >
+                    <field name="lot_id" readonly="1" />
+                </xpath>
+            </field>
+        </record>
+
+        <record id="purchase_order_line_lot_form" model="ir.ui.view">
+            <field name="model">purchase.order.line</field>
+            <field name="inherit_id" ref="purchase.purchase_order_line_form2" />
+                <field name="arch" type="xml">
+                    <xpath expr="//field[@name='product_id']" position="after">
+                    <field name="lot_id" readonly="1" />
+                </xpath>
+            </field>
+        </record>
+
+        <record id="view_purchase_order_filter" model="ir.ui.view">
+            <field name="model">purchase.order</field>
+            <field name="inherit_id" ref="purchase.view_purchase_order_filter" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='name']" position="after">
```

Comment:
`<field name="name"></field>` should suffice

Reply:
I don't know how to do this... in previous PR's I use that approach, and the reviewers suggest me to use xpath.

I think this depends on preference and it will work the same way.